packages/config/config/devices/0x0460/qnsw-001P16.json

```diff
@@ -56,11 +56,15 @@
 		{
 			"#": "17",
 			"$import": "~/templates/master_template.json#state_after_power_failure_prev_off",
-			"label": "O1: State After Power Failure"
+			"label": "O1: State After Power Failure",
+			"minValue": 0,
+			"maxValue": 1,
+			"unsigned": true
```

Comment:
```suggestion
```
These are unnecessary. The imported template already contains everything that's needed.